Makefile

```diff
@@ -57,17 +65,37 @@ run-local: install-apis ## Install CRDs and run provisioner locally
 	kubectl create namespace rukpak-system
 	$(Q)go run provisioner/k8s/main.go
 
+deploy: install-apis ## Deploy the operator to the current cluster
+	kubectl apply -f provisioner/k8s/manifests
+
+run: build-local-container load-image deploy
+
+## --------------------------------------
+## Build and Load
+## --------------------------------------
+
 # Binary builds
-GO_BUILD := $(Q)go build
+.PHONY: build bin/k8s bin/unpack
 
-build: bin/k8s
+GO_BUILD := $(Q)go build
+build: bin/k8s bin/unpack
 
 bin/k8s:
 	CGO_ENABLED=0 go build -o $@ ./provisioner/k8s
 
 bin/unpack:
 	CGO_ENABLED=0 go build -o $@ ./cmd/unpack/...
 
+build-local-container: ## Builds provisioner container image locally
```

Comment:
is there any interest in having a local.Dockerfile which just takes the built binaries from the local machine and puts them in a container instead, like combo has? The build time for the container is pretty slow locally, wondering if it makes sense to avoid that, or if we only want to test one specific way as @joelanford is suggesting in https://github.com/operator-framework/rukpak/pull/85/files#r817721452.

Reply:
That would be my preference but I thought we ran into some opposition when we were talking about that last week? Maybe it's fine for now and we can track the want/need for a dev.Dockerfile?

Reply:
Can we have the _real_ build:
1. Enforce a specific version of Go
2. Ensure the locally built binary is up-to-date, including all the necessary build flags that we would want in our production image?
3. Copy that binary into the image.